docs/dev-environments/ethereum/besu/cleanup.sh

```diff
@@ -13,10 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+exit_status=0
 function error_exit()
 {
     echo "[Error]: " ${*}
-    exit 1
+    exit_status=1
```

Comment:
This function doesn't exit. Can we rename this function in that case. Else let's the function exit on error